### PR TITLE
Update mods.json

### LINKED ISSUE #31: Request addition of MST_Extra mod
Name: MST_Extra_Mod for Cataclysm: Dark Days Ahead
Url: https://github.com/chaosvolt/MST_Extra_Mod
Author: chaosvolt
Homepage: https://github.com/chaosvolt/MST_Extra_Mod
Mod not found in version: 1.6.3

### PR DESCRIPTION
**Pull request by DazedNConfused-**
Closes #31

### CODE REVIEW

**Comment by Thogsloc** [comment]
updated mods.json doesn't work

File: data/mods.json [hunk 1]
```diff
@@ -347,5 +347,16 @@
...
-  }
+  },
+  {
+    "type": "direct_download",
+    "mod_type": "SUPPLEMENTAL",
+    "ident": "MST_Extra",
+    "name": "MST_Extra_Mod for Cataclysm: Dark Days Ahead",
+    "description": "the official followup to the now-obsolete More Survival Tools mod, Adds assorted additional content and changes that improve quality of life when playing innawoods",
+    "author": "chaosvolt",
+    "size": 527112
```
Comment (Tranberry) [inline data/mods.json:358]: trailing `,` needed

File: data/mods.json [hunk 2]
```diff
@@ -347,5 +347,16 @@
...
+    "url": "https://github.com/chaosvolt/MST_Extra_Mod/archive/refs/heads/master.zip",
+    "homepage": "https://github.com/chaosvolt/MST_Extra_Mod"
+  },
```
Comment (Tranberry) [inline data/mods.json:361]: no trailing `,` here

**Review by Tranberry — COMMENTED** [review]
the `json` is invalid, but it's easy to fix.

**Pull request merged**